engine/window/Makefile

```diff
@@ -16,7 +16,7 @@ windowDepends := $(patsubst %.o, %.d, $(call rwildcard,$(windowBinDir)/,*.o))
 windowLibs := $(vendorDir)/glfw/build/src/libglfw3.a
 
 linkFlags += -l utils
-compileFlags += -I $(vendorDir)/glfw/include
+compileFlags += -I $(vendorDir)/glfw/include -I $(vendorDir)/vulkan/include
```

Comment:
This is undoing the work of the last PR in severing dependencies. The contents of `GetWindowSurface` should be happening inside of the render module, otherwise Window must require Vulkan, which creates a two-way dependency.

Reply:
I've just cleaned this up and pushed a commit for the change